EfEnumToLookupTests/Tests/TestStuff.cs

```diff
@@ -19,8 +19,11 @@ public void SetUp()
 			// Using setup rather than teardown to make it easier to inspect the database after running a test.
 			using (var context = new MagicContext())
 			{
-				context.Database.Delete();
-			}
+                if(context.Database.Exists())
```

Comment:
It did work most of the times with out the exists check but sometimes it crashed with the message can't delete a database that did not exist. So that was why I added it.  I was using VS 2015. But that should not made a difference. I believe the exists check is not needed on a build server but I think VS sometimes gets into a state where this fails.